packages/now-build-utils/package.json

```diff
@@ -34,10 +36,12 @@
     "fs-extra": "7.0.0",
     "glob": "7.1.3",
     "into-stream": "5.0.0",
+    "jest": "24.9.0",
     "minimatch": "3.0.4",
     "multistream": "2.1.1",
     "node-fetch": "2.2.0",
     "semver": "6.1.1",
+    "ts-jest": "24.1.0",
```

Comment:
What do you think about adding `ts-jest` to the root package.json so we can write all our tests with typescript?